[EDU-4186] feat: add CRL digital certificates (#843)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx

---

## Certificate Revocation List (CRL)

> A **Certificate Revocation List (CRL)** is a list of digital certificates revoked before their expiration date. You can add it in Azion, and it works as a blocking list, stating those certificates can't be trusted. The issuing certificate authority (CA) signs this list to guarantee its security.

Reviewer: ```suggestion
A **Certificate Revocation List (CRL)** is a list of digital certificates revoked before their expiration date. It works as a block list that you can add it to your account to state those certificates can't be trusted. The issuing certificate authority (CA) signs this list to guarantee its security.
```

> The CRL works when associated with a [mTLS CA Certificate](/en/documentation/products/build/edge-application/domains/mtls/). Therefore, you must have mTLS enabled in the domain you want to associate it with.

Reviewer: ```suggestion
The CRL works when associated with an [mTLS CA Certificate](/en/documentation/products/build/edge-application/domains/mtls/). Therefore, you must have mTLS enabled in the domain you want to associate it with.
```

The CRL is automatically validated before the creation is successful in the Azion Edge Computing Platform. The management, including creation, of a CRL is available via [Azion API](https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248).

If you have multiple CRLs, you must add one list at a time, and separate each certificate with a line break.

> <Button href="https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248" text="create CRL via API" variant="secondary"></Button>

Reviewer: ```suggestion
<Button href="https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248" text="create CRL via API" variant="secondary" />
```

Assim fica mais clean 🤓

You can [associate a CRL with a domain](https://api.azion.com/#d3b79af9-e708-4a4f-9bc6-d2df3700c502) via API.

:::note

> You can't delete a CRL currently associated with a domain.

Reviewer: ```suggestion
You can't delete a CRL that is associated with a domain. You must first remove the CRL from the domain to delete it.
```

Notei que existe o delete então melhor esclarecer
